### DISCUSSION TITLE
Querying "menu" returns empty string

### DISCUSSION THREAD

**Discussion by jhlindnerberlin** [Shopify/storefront-api-feedback#226] [Help]
After reading everything I could find in docs, google, here and everywhere - graphql endpoint is not returning anything but an empty string when I try to load a menu by handle. For anything else I receive data as expected - for products, collections etc. for instance.
The access scope is fine - granted all privileges for testing purposes. 

I am using cURL in a PHP script. 

```
$data = "{
		\"query\":\"{
			menu(handle: \"main-menu\"){
				items {
					title
				}
			}
		}\"
	}";
```

The response is empty. HTTP_CODE is 200. If I remove the argument handle, I get an error as response: {"errors":[{"message":"Field 'menu' is missing required arguments: handle","locations":[{"line":3,"column":4}],"path":["query","menu"],"extensions":{"code":"missingRequiredArguments","className":"Field","name":"menu","arguments":"handle"}}]}

So it looks like it should work when I pass an argument. I've tried with several menu handles. Double checked the handles with admin UI / json. 

I've tried tons of different things, with headerMenu, different string representations... no chance. It's always empty. 

Could anyone give me a hint? Thanks in advance!

**Comment by johncraigcole** [comment]
Not seeing any problems with the following cURL request from the terminal

```
curl -X POST \                                                                                                                    ─╯
  https://yourdomain.myshopify.com/api/2021-07/graphql.json \
  -H 'Content-Type: application/json' \
  -H 'X-Shopify-Storefront-Access-Token:sometoken' \
  -d '{
    "query": "{menu(handle:"main-menu") {
                items {
                        title
                }
        }}"
  }'
```

Have you installed the Shopify GraphiQL app and tried the query there just to validate its returning anything?
